### ISSUE TITLE
Не работает действие обновления через интент

### ISSUE THREAD

**Issue by alex3696** [dukei/any-balance-providers#1063]
Galaxy s9+, android 9.0 
С помощью macrodroid запускаю интент com.dukei.android.apps.anybalance.appwidget.action.MY_OWN_WIDGET_UPDATE - обновление баланса не происходит, на android 8 работало. Обе программы исключены из настроек энергосбережения, сбрасывать настройки пробовал.
Если это сделать сложно, прошу сделать опцию обновления баланса при подключении питания с задержкой минут в 5.

**Comment by dukei** [comment]
Да, из-за требований гугл запуск через интент был изменен. В ближайшие
недели обновим плагин к таскер и уточним, как запускать этот интент с
помощью других программ.

пн, 1 апр. 2019 г. в 21:58, alex3696 <notifications@github.com>:
